yt_dlp/extractor/kaltura.py

```diff
@@ -226,7 +311,70 @@ def _get_video_info(self, video_id, partner_id, service_url=None):
             },
         ]
         return self._kaltura_api_call(
-            video_id, actions, service_url, note='Downloading video info JSON')
+            video_id, actions, service_url, note='Downloading video info JSON (Kaltura html5 player)')
+
+    def _get_video_info_kwidget(self, video_id, partner_id, service_url=None):
+        widget_id = (partner_id if '_' in partner_id else ('_%s' % partner_id))
```

Comment:
same as above.

Also maybe this makes sese as a function?

Reply:
Done